scripts/processAndDownloadNFTs.ts

```diff
@@ -133,23 +133,28 @@ async function processNfts() {
 	const openseaData = await fetchOpenseaFromUsers(NFT_OWNER_ADDRESSES)
 	console.log(openseaData)
 	const output = `
-
-const nftData: [string, string, string, string | null, string | null][] = ${JSON.stringify(openseaData.map((x) => [addressString(x.address), x.data.name + (x.data.hidden ? '[hidden]' : '') + +(x.data.featured ? '[featured]' : ''), x.data.symbol, x.data.protocol, x.data.logoURI]), null, 1)}
+import * as nftData from './nftMetadata.json';
 export type NftDefinition = {
-    name: string,
-    symbol: string,
-    protocol: string | undefined,
-    logoURI: string | undefined,
+	name: string,
+	symbol: string,
+	protocol: string | undefined,
+	logoURI: string | undefined,
 }
 export const nftMetadata = new Map<string, NftDefinition>(
-    nftData.map( ([address, name, symbol, protocol, logoURI] ) => [address, {
-      logoURI: logoURI === null ? undefined : logoURI,
-      protocol: protocol === null ? undefined : protocol,
-      name,
-      symbol: symbol,
-    }])
+	nftData.reduce(( acc, [address, name, symbol, protocol, logoURI] ) => {
+		if (address === null) return acc
+		return acc.concat([
+			[address, {
+				name: name === null ? 'undefined' : name,
+				symbol: symbol === null ? 'undefined' : symbol,
+				logoURI: logoURI === null ? undefined : logoURI,
+				protocol: protocol === null ? undefined : protocol,
+			}]])
+	}, [] as [string, NftDefinition][])
 )
 `
+	const nftData = JSON.stringify(openseaData.map(( x ) => [addressString(x.address), x.data.name + (x.data.hidden ? '[hidden]' : '') + +(x.data.featured ? '[featured]' : ''), x.data.symbol, x.data.protocol, x.data.logoURI]), null, '\t')
+	fs.writeFileSync(`${OUTPUT_SRC_DIR}/nftMetadata.json`, nftData, 'utf-8')
 	fs.writeFileSync(`${OUTPUT_SRC_DIR}/nftMetadata.ts`, output, 'utf-8')
 }
```

Comment:
Same general feedback as for the contract metadata, but applied to NFT metadata.